src/Microsoft.DotNet.Cli.Utils/PathUtility.cs

```diff
@@ -261,6 +261,32 @@ public static string GetPathWithDirectorySeparator(string path)
             }
         }
 
+        public static string RemoveExtraPathSeparators(string path)
+        {
+            if (string.IsNullOrEmpty(path))
+            {
+                return path;
+            }
+
+            var components = path.Split(Path.DirectorySeparatorChar);
```

Comment:
Not necessarily. But in the FilePath api (in SlnFile) we always convert to OS specific path separator. This utility method shouldn't know about VS so I just made it always deal in OS specific paths.